server/api/v1/controllers/quotesApiController.js

```diff
@@ -0,0 +1,279 @@
+"use strict";
+
+const quotesManager = require("../../../../backend/quotes/quotes-manager");
+const moment = require("moment");
+
+exports.getQuotes = async function(req, res) {
+    const quotes = await quotesManager.getAllQuotes();
+
+    const formattedQuotes = quotes.map(q => {
+        return formatQuote(q);
+    });
+
+    res.json(formattedQuotes);
+};
+
+exports.getQuote = async function(req, res) {
+    let { quoteId } = req.params;
+
+    if (quoteId == null) {
+        return res.status(400).send({
+            status: "error",
+            message: "No quoteId provided"
+        });
+    }
+
+    quoteId = parseInt(quoteId);
+    if (isNaN(quoteId)) {
+        return res.status(400).send({
+            status: "error",
+            message: "Invalid quoteId provided"
+        });
+    }
```

Comment:
The logic for these two checks are reused for a number of the endpoints here.  Good candidate to bump out into a function to keep us DRY.

Perhaps something like
```js
function validateQuoteId(quoteId, res) {
   if (quoteId == null) {
        res.status(400).send({
            status: "error",
            message: "No quoteId provided"
        });
        return false;
    }

    quoteId = parseInt(quoteId);
    if (isNaN(quoteId)) {
        res.status(400).send({
            status: "error",
            message: "Invalid quoteId provided"
        });
        return false;
    }
    
    return quoteId;
}

// elsewhere...
quoteId = validateQuoteId(quoteId, res);
if(!quoteId) {
  return;
}
```

Reply:
Fixed